shared/src/test/diff/pretyper/ucs/DualOption.mls

```diff
@@ -1,13 +1,13 @@
 :NewDefs
 :PreTyper
 
-class Some[T](value: T)
-module None
-type Option[T] = Some[T] | None
```

Comment:
Why this change? It doesn't work with the type alias?

Reply:
It works. I just wanted to see if there's any differences if I use abstract classes. I'll keep both versions in different test files.